.github/workflows/publish.yml

```diff
@@ -36,6 +36,12 @@ jobs:
         with:
             dotnet-version: '7.0.x'
 
+      - name: Install Nuget
```

Comment:
Is that the right name?

Reply:
Also, why use that pacakge instead of https://github.com/actions/setup-dotnet#caching-nuget-packages?
Upgrade setup-dotnet to @v3 and it will support caching out of the box iiuc

Reply:
So it isn't obvious that install dotnet installs nuget. It does install 'dotnet nuget' which may or may not be the same thing. They don't have the same usage, and I can't easily find an equivalent to "nuget list" (dotnet nuget list is different). With that said, I don't mind upgrading to v3